<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>306</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>306</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1a1a1a;
color: #ffffff;
font: 9pt &quot;Lato Black&quot;</string>
  </property>
  <widget class="QFrame" name="title_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato Black</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffbe00;
font: 14pt &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>HISTORY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aotButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>11</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:#7f7f7f;
	background:transparent;
	border:3px solid #7f7f7f;
}
QPushButton:checked{
	color: #242424;
	background: #ffffff;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>ALWAYS ON TOP</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>450</width>
     <height>206</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background:transparent;
	border: 3px solid #000000
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="history_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>3</y>
      <width>417</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView{
	background: transparent;
	border:none;
	font: 9pt &quot;Lato&quot;
}
QScrollBar{
	width: 10px;
	background: #252525;
	border:none;
	margin: 0px 0px 0px 0px;
}
QScrollBar::handle{
	background: #1a1a1a;
}
QScrollBar::add-line, QScrollBar::sub-line{
	border:none;
	background: none;
}
QScrollBar::add-page, QScrollBar::sub-page{
	border:none;
	background: none;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>256</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
